### PR TITLE
Parse event data as CSV, and build a Python dict

### PR DESCRIPTION
**Pull request by onyb**
- Remove use of Numpy's genfromtxt()
- Support for JSON dumps
- Added documentation of the data set

Improves upon pull request #5 by @Tontonis.

### CODE REVIEW

**Comment by Tontonis** [comment]
Much neater than my attempt, cool!

**Comment by Tontonis** [comment]
Pull request merged.

**Comment by RaoOfPhysics** [comment]
@Tontonis: OT, but thought you should know I'm in the Vidyo room we usually use.

File: station_data.py [hunk 1]
```diff
@@ -7,6 +6,7 @@
 from pprint import pprint
 from StringIO import StringIO
 from numpy import genfromtxt
```
Comment (kinow) [inline station_data.py:8]: I think this import can be removed too

**Comment by kinow** [comment]
Tested locally, working fine. Code looking good too :smile: as we are just experimenting I think it's fine to skip tests for now, but with a good code like this we can soon start writing tests if we settle down for a format.

**Comment by onyb** [comment]
@kinow: We can manage with some simple asserts for now. We might need better tests once we have a detailed specification for the format. In case you haven't seen, Frank has put up a rough map of the data structure on Gitter.

File: station_data.py (hunk 1 above)
Comment (onyb) [inline station_data.py:8]: Thanks for pointing out. I will include this in my next PR.

**Pull request merged**